packages/2d/src/lib/components/Video.ts

```diff
@@ -159,16 +226,55 @@ export class Video extends Media {
     return video;
   }
 
-  protected override draw(context: CanvasRenderingContext2D) {
+  protected lastFrame: HTMLImageElement | null = null;
+
+  protected async serverSeekedVideo(): Promise<HTMLImageElement> {
+    const video = this.video();
+    const time = this.clampTime(this.time());
+    const duration = this.getDuration();
+
+    video.playbackRate = this.playbackRate();
+
+    if (this.lastFrame && this.lastTime === time) {
+      return this.lastFrame;
+    }
+
+    const fps = this.view().fps() / this.playbackRate();
```

Comment:
the resulting fps number could be a non-integer (e.g. for fps=30, playbackrate=1.1), not sure if this could cause issues

Reply:
This is supported by ffmpeg so it should be fine.